src/components/Admin/Messenger.jsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+
+import React, { useState } from "react";
+import Title from "./Title";
+
+const ADDRESSEES = ["volunteers", "mentors", "judges"];
+
+const Message = () => {
+  const [addressees, setAddressee] = useState([]);
+  const [subjectText, setSubjectText] = useState("");
+  const [messageBody, setMessageBody] = useState("");
+
+  const ToSelector = ({ selected, setSelected, allSelectorNames }) => {
```

Comment:
Don't make another component

Reply:
Check the admissions page for the implementation. it will reduce ur code significantly